docs/changes/12199.other.rst

```diff
@@ -0,0 +1 @@
+Add missing package ``packaging``  to ``setup.cfg``.
```

Comment:
It is not missing from the last release, but rather an unreleased patch now requires it, so we need to reword. 😸 

```suggestion
``astropy`` now requires ``packaging``.
```